src/index.html

```diff
@@ -11,7 +11,178 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="test-form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="text"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="text"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label for="old">
+            How old are You?
+            <input
+              type="number"
+              name="number"
+              value="12"
+              min="1"
+              max="100"
+              id="old"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            I accept the term of the agreement
+            <input
+            type="checkbox"
+            name="checkbox"
+            required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="input">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Password:
+            <input
+            type="password"
+            name="password"
+            minlength="6"
+            maxlength="10"
+            required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label>
+            Do you love cats?
```

Comment:
```suggestion
          Do you love cats?

          <label>
```
This text should be outside the first label because now clicking on the text `Yes` option selects - it's the wrong behavior